content/v2.0/reference/syntax/annotated-csv/extended.md

```diff
@@ -0,0 +1,360 @@
+|:----------                    |:-----------------------                 |
+| [measurement](#measurement)   | Column is the **measurement**           |
+| [tag](#tag)                   | Column is a **tag**                     |
+| [dateTime](#datetime)         | Column is the **timestamp**             |
+| [field](#field)               | Column is a **field**                   |
+| [ignored](#ignored)           | Column is ignored                       |
+| [string](#string)             | Column is a **string field**            |
+| [double](#double)             | Column is a **float field**             |
+| [long](#long)                 | Column is an **integer field**          |
+| [unsignedLong](#unsignedlong) | Column is an **unsigned integer field** |
+| [boolean](#boolean)           | Column is a **boolean field**           |
+
+#### measurement
+Indicates the column is the **measurement**.
+
+#### tag
+Indicates the column is a **tag**.
+The **column label** is the **tag key**.
+The **column value** is the **tag value**.
+
+#### dateTime
+Indicates the column is the **timestamp**.
+`time` is as an alias for `dateTime`.
+If the [timestamp format](#supported-timestamp-formats) includes a time zone,
+the parsed timestamp respects the time zone.
+By default, all timestamps are UTC.
+You can also use the [`#timezone` annotation](#timezone) to adjust timestamps to
+a specific time zone.
+
+{{% note %}}
+There can only be **one** `dateTime` column.
+{{% /note %}}
+
+The `influx write` command converts timestamps to [Unix timestamps](/v2.0/reference/glossary/#unix-timestamp).
+Append the timestamp format to the `dateTime` datatype with (`:`).
+
+```csv
+#datatype dateTime:RFC3339
+#datatype dateTime:RFC3339Nano
+#datatype dateTime:number
+#datatype dateTime:2006-01-02
+```
+
+##### Supported timestamp formats
+
+| Timestamp format | Description       | Example                          |
+|:---------------- |:-----------       |:-------                          |
+| **RFC3339**      | RFC3339 timestamp | `2020-01-01T00:00:00Z`           |
+| **RFC3339Nano**  | RFC3339 timestamp | `2020-01-01T00:00:00.000000000Z` |
+| **number**       | Unix timestamp    | `1577836800000000000`            |
+
+{{% note %}}
+If using the `number` timestamp format and timestamps are **not nanosecond Unix timestamps**,
+use the [`--precision` flag](/v2.0/reference/cli/influx/write/#flags) with the
+`influx write` command to specify the timestamp precision.
+{{% /note %}}
+
+##### Custom timestamp formats
+To specify a custom timestamp format, use timestamp formats as described in the
+[Go time package](https://golang.org/pkg/time).
+For example: `2020-01-02`.
+
+#### field
+Indicates the column is a **field** and auto-detects the field type.
+The **column label** is the **field key**.
+The **column value** is the **field value**.
+
+#### ignored
+The column is ignored and not written to InfluxDB.
+
+#### Field types
```

Comment:
field types

Reply:
Curious, why would you specify a field type if the field type is automatically detected?

Reply:
@kelseiv

> field types

I capitalized "Field" because this isn't a literal datatype. This section includes annotations that explicitly specify the field type.

> Curious, why would you specify a field type if the field type is automatically detected?

It's not automatically detected (I was corrected by @sranka). The column value is copied as-is to the line protocol so it's likely numeric values won't be written in the correct format since they require a suffix in line protocol to determine their type (for example: `1i` is an interger, `1u` is a uinterger, and `1` is a float).